blog/_posts/2021-07-06-distributed-rdataframe.md

```diff
@@ -0,0 +1,105 @@
+For example, backends for Dask and AWS Lambda have already been implemented and demonstrated at different conferences
+[[1]](#dasksummit)[[2]](#hips21). They will be made available in future ROOT releases.
+
+Great attention is being put towards ensuring that running the RDataFrame computation graph scales well across
+multiple computing nodes and different backend implementations. This has been shown since the first stages of the
+development of this package with a real use case analysis running on a Spark cluster [[3]](#europar19). More recently, a benchmark
+based on CERN open data has shown promising scaling performance with both Spark and Dask [[4]](#pyhep21):
+
+![Scaling of distributed RDataFrame running the dimuon analysis on 100 times the original data. Dask and Spark backends compared]({{ '/assets/images/distrdf_dimuon_scaling_dask_spark_pyhep2021.png' | relative_url }})
+
+All it takes to steer an existing RDataFrame application to distributed resources is to use the correct RDataFrame
+instance and provide it with the arguments needed to connect to the remote cluster. Let's see an example of connection
+to a Dask cluster:
+
+```python
+from dask.distributed import Client
+import ROOT
+
+# Point RDataFrame calls to the Dask specific RDataFrame
+RDataFrame = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame
+
+# Create the Client object to connect to the Dask cluster
+# See the Dask documentation for all the options available
+client = Client("DASK_SCHEDULER_ADDRESS")
+
+# It still accepts the same constructor arguments as traditional RDataFrame
+# And supports some extra keyword arguments
+df = RDataFrame("mytree", "myfile.root", npartitions = 8, daskclient = client)
+```
+
+In this example the `npartitions` parameter tells the RDataFrame into how many ranges of entries the input dataset should
+be split. Each range will then correspond to a task on a node of the cluster. The `daskclient` parameter receives the
+object needed to connect to the Dask scheduler. All the options are available in the
+[Dask documentation](https://distributed.dask.org/en/latest/client.html). The equivalent object for the Spark framework
+is called [SparkContext](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.SparkContext.html) and in
+general every backend will have its own way to connect to a cluster of nodes.
+
+Once the correct `RDataFrame` object has been created, there is no need to modify any other part of the program.
+
+## Conclusions
+
+Distributed RDataFrame enables large-scale interactive data analysis with ROOT. This Python layer on top of RDataFrame
+allows to steer C++ computations on a set of computing nodes and returning the final result directly to the user, so
+that the entire analysis can be run within the same application. It is available as an experimental feature since ROOT
+6.24 with the support for running on a Spark cluster, with more backends coming in future ROOT releases. Try it with our
```

Comment:
```suggestion
6.24 with the support for running on a Spark cluster, with more backends in the works: Dask is already available in our nightly builds. Try it with our
```

Reply:
Ok so while it is true that the Dask backend could be considered "done" in a sense, it's not in master so it's not really in our nightly builds. I'm not sure it's good to write it that way

Reply:
"will soon be available in our nighly builds"?